main.py

```diff
@@ -171,7 +172,7 @@ def input_kontaktdaten_key(
             print(f"\n{str(exc)}\n")
 
 
-def run_search_interactive(kontaktdaten_path, check_delay):
+def run_search_interactive(kontaktdaten_path, check_delay,telegram=None):
```

Comment:
`telegram` wird immer übergeben, oder? Wozu der Default?

```suggestion
def run_search_interactive(kontaktdaten_path, check_delay, telegram):
```

Reply:
War mir nicht sicher von wo genau das alles aufgerufen wird, mit dem default wäre es auf jeden Fall weiterhin gegangen